### PR TITLE
Fix benchmarks on netcoreapp2.0

### PR DESCRIPTION
**Pull request by rynowak**

### CODE REVIEW

File: test/Microsoft.AspNetCore.Server.Kestrel.Performance/Microsoft.AspNetCore.Server.Kestrel.Performance.csproj [hunk 1]
```diff
@@ -8,6 +8,12 @@
     <ServerGarbageCollection>true</ServerGarbageCollection>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsPackable>false</IsPackable>
+    
+    <!-- 
+    This is required right now for BenchmarkDotNet to restore the benchmarks. It is NOT ok to move this into
+    an imported .props file. See https://github.com/dotnet/BenchmarkDotNet/issues/406
```
Comment (natemcmaster) [inline test/Microsoft.AspNetCore.Server.Kestrel.Performance/Microsoft.AspNetCore.Server.Kestrel.Performance.csproj]: 🤔 are they parsing XML directly?

Comment (rynowak) [inline test/Microsoft.AspNetCore.Server.Kestrel.Performance/Microsoft.AspNetCore.Server.Kestrel.Performance.csproj]: Looks like it.

Comment (davidfowl) [inline test/Microsoft.AspNetCore.Server.Kestrel.Performance/Microsoft.AspNetCore.Server.Kestrel.Performance.csproj]: Yes, I think so.

**Comment by rynowak** [comment]
0ab49f4977a16e38b6db80731f4c2ab2615b4bd0

**Pull request closed without merging**